### ISSUE TITLE
bug:  `replaceAll` is not a function

### ISSUE THREAD

**Issue by MaxumCSmith** [simonhaenisch/md-to-pdf#173]
**Context:**

5 minutes ago a new version was published. I've been running the following command successfully ( until 5 minutes ago ).

```pwsh
npx md-to-pdf .\cash_flow_discussion.md
  × generating PDF from .\\cash_flow_discussion.md
    → (0 , path_1.join)(...).replaceAll is not a function
ListrError: Something went wrong
    at C:\Users\mcs22\AppData\Local\npm-cache\_npx\55158e48eb5c59f7\node_modules\listr\index.js:102:18
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async main (C:\Users\mcs22\AppData\Local\npm-cache\_npx\55158e48eb5c59f7\node_modules\md-to-pdf\dist\cli.js:121:5) {
  errors: [
    TypeError: (0 , path_1.join)(...).replaceAll is not a function
        at generateOutput (C:\Users\mcs22\AppData\Local\npm-cache\_npx\55158e48eb5c59f7\node_modules\md-to-pdf\dist\lib\generate-output.js:31:70)
        at processTicksAndRejections (internal/process/task_queues.js:95:5)
        at async convertMdToPdf (C:\Users\mcs22\AppData\Local\npm-cache\_npx\55158e48eb5c59f7\node_modules\md-to-pdf\dist\lib\md-to-pdf.js:61:20)
  ],
  context: [Object: null prototype] {}
}
```

- version: whatever was prior to 5 minutes ago, presumably this tag - https://github.com/simonhaenisch/md-to-pdf/releases/tag/v5.2.1
- platform (Unix, macOS, Windows): Windows
- node version: 14.19.1

**Describe the bug:**

See text above.

**Comment by MaxumCSmith** [comment]
This tag broke it. https://github.com/simonhaenisch/md-to-pdf/releases/tag/v5.2.2

**Comment by MaxumCSmith** [comment]
This workaround work for me.

`npx md-to-pdf@5.2.0 .\cash_flow_discussion.md`

**Comment by simonhaenisch** [comment]
Oh yup that's a regression I'll publish a fix now... forgot that `replaceAll` is not available in Node 14 yet.

**Comment by simonhaenisch** [comment]
Fixed in 018b52e.

**Comment by MaxumCSmith** [comment]
🙇

**Comment by simonhaenisch** [comment]
Just published version `5.2.3` with the fix.

**Comment by MaxumCSmith** [comment]
Tested. It works. Thank you.
